source/core/parameters_lagrangian.cc

```diff
@@ -409,44 +433,48 @@ namespace Parameters
           prm.get_integer("inserted number of particles at each time step");
         insertion_frequency = prm.get_integer("insertion frequency");
 
-        std::vector<int> axis_order =
-          convert_string_to_vector<int>(prm, "insertion direction sequence");
-        if (axis_order.size() == 2)
-          axis_order.resize(3);
-
-        direction_sequence.reserve(3);
-        direction_sequence.push_back(axis_order[0]);
-        direction_sequence.push_back(axis_order[1]);
-        direction_sequence.push_back(axis_order[2]);
-
-        const std::vector<std::string> point_coordinates_list(
+        // Clear and add:
+        const std::vector<std::string> clearing_box_point_coordinates_list(
           Utilities::split_string_list(
-            prm.get("insertion box points coordinates"), ":"));
+            prm.get("clearing box points coordinates"), ":"));
 
-        std::vector<double> point_coord_temp_1 = Utilities::string_to_double(
-          Utilities::split_string_list(point_coordinates_list.at(0)));
-        std::vector<double> point_coord_temp_2 = Utilities::string_to_double(
-          Utilities::split_string_list(point_coordinates_list.at(1)));
+        std::vector<double> clearing_point_coord_temp_1 =
+          Utilities::string_to_double(Utilities::split_string_list(
+            clearing_box_point_coordinates_list.at(0)));
+        std::vector<double> clearing_point_coord_temp_2 =
+          Utilities::string_to_double(Utilities::split_string_list(
+            clearing_box_point_coordinates_list.at(1)));
 
-        if (point_coord_temp_1.size() == 2 && point_coord_temp_2.size() == 2)
+        if (clearing_point_coord_temp_1.size() == 2 &&
+            clearing_point_coord_temp_2.size() == 2)
           {
-            point_coord_temp_1.resize(3);
-            point_coord_temp_2.resize(3);
+            clearing_point_coord_temp_1.resize(3);
+            clearing_point_coord_temp_2.resize(3);
           }
-        for (int i = 0; i < 3; ++i)
+
+        for (unsigned int i = 0; i < 3; ++i)
           {
-            insertion_box_point_1[i] = point_coord_temp_1.at(i);
-            insertion_box_point_2[i] = point_coord_temp_2.at(i);
+            clear_box_point_1[i] = clearing_point_coord_temp_1.at(i);
+            clear_box_point_2[i] = clearing_point_coord_temp_2.at(i);
           }
 
-        distance_threshold = prm.get_double("insertion distance threshold");
-        insertion_maximum_offset = prm.get_double("insertion maximum offset");
-        seed_for_insertion       = prm.get_integer("insertion prn seed");
+        list_of_input_files =
+          convert_string_to_vector<std::string>(prm, "list of input files");
 
-        initial_vel = entry_string_to_tensor3(prm, "initial velocity");
-        initial_omega =
-          entry_string_to_tensor3(prm, "initial angular velocity");
+        //// File:
```

Comment:
Avoid the use of ////
Make comments block with /* */ if you need a different comment style at a location.